### PR TITLE
refactor(Calcified): use `Int`s instead of `Byte`s for port numbers

### LINKED ISSUE #4: Change Calcified hardware ports to use Ints instead of Bytes, for better Java compatability

### PR DESCRIPTION
**Pull request by dr-hextanium**
in order for better Java compatibility (remove the need for casting to byte which was noisy), resolves #4

### CODE REVIEW

**Comment by dr-hextanium** [comment]
i see what i messed up here

**Comment by dr-hextanium** [comment]
TODO: remove the gitconfig stuff and then it should actually be fine

File: .gitmodules
Comment (Froze-N-Milk) [inline .gitmodules:1]: revert changes to https and change all to ssh

**Review by Froze-N-Milk — CHANGES_REQUESTED** [review]
1. See comments on .gitmodules
2. Resolve conflict in Calcified
3. Update java overview to no longer cast to byte

**Review by Froze-N-Milk — APPROVED** [review]

**Pull request merged**